platform-workflow-module/src/test/java/com/softicar/platform/workflow/module/workflow/task/WorkflowTaskNotificationSubmitterTest.java

```diff
@@ -78,4 +79,13 @@ private String getMyTasksPageUrl() {
 
 		return new PageUrlBuilder<>(WorkflowTaskPage.class, moduleInstance).build().toString();
 	}
+
+	private AGLocalization getEnglishLocalization() {
+
+		return AGLocalization.TABLE//
+			.createSelect()
+			.where(AGLocalization.NAME.equal("International"))
```

Comment:
not sure if this is a good hard-coding, couldn't we add a variable somewhere?

Reply:
created a constant